docker-compose.yml

```diff
@@ -2,6 +2,7 @@ version: '3.1'
 services:
   mysql:
     image: mysql:5.6
+    platform: linux/amd64
```

Comment:
This allows it all to run on apple silicon.

Reply:
@kschelonka note just hit this with herraj, since we digest pinning in most repos we need to make sure this line is always in our docker composes